package.json

```diff
@@ -26,16 +26,18 @@
     "express-rate-limit": "1.0.2",
     "extract-text-webpack-plugin": "0.8.0",
     "file-loader": "0.8.4",
+    "google-spreadsheet": "^1.0.1",
     "habitat": "0.4.1",
-    "hatchet": "0.3.2",
```

Comment:
removing this since we are not using `hatchet` in the code anymore

@ScottDowne and I hold different opinions on whether or not to leave this line in `package.json`... and neither of us knew what the best practice is. :stuck_out_tongue: @Pomax thoughts?

His view:
``` 
I think it's just about expectations of what package.json is used for being consistent
If in a year time I was tasked with sending an email from mozfest site, I wouldn't have expected to see a package missing, I would expect to go to .env for that sort of config
but, it's technically not going to break anything, so it's just a matter of expectations and best practice
```

My preference is to keep the codebase up-to-date -- remove code and modules that aren't being used anymore.  If for any reason we want to get "old" code back we use commit history and GitHub tickets to recall our lost memory.  And I also think in a year time we are probably gonna switch to use another module (vs `hatchet`) :hamster: